akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -391,6 +391,20 @@ class TestKit(system: ActorSystem) {
   @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.12")
```

Comment:
Should we update this deprecation warning? (`since = "2.5.13"`)

Reply:
Good point will address it thanks !